qdrant-landing/content/articles/product-quantization.md

```diff
@@ -0,0 +1,211 @@
+that the performance might be reduced. However, memory usage might be reduced drastically as 
+well. As usual, we did some benchmarks to give you a brief understanding of what you may expect.
+
+Again, we reused the same pipeline as in [the other benchmarks we published](/benchmarks). We
+selected [Arxiv-titles-384-angular-no-filters](https://github.com/qdrant/ann-filtering-benchmark-datasets)
+and [Glove-100](https://github.com/erikbern/ann-benchmarks/) datasets to measure the impact
+of Product Quantization on precision and time. Both experiments were launched with $ EF = 128 $. 
+The results are summarized in the tables:
+
+#### Glove-100
+
+<table>
+   <thead>
+      <tr>
+         <th></th>
+         <th>Original</th>
+         <th>1D clusters</th>
+         <th>2D clusters</th>
+         <th>3D clusters</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <th>Mean precision</th>
+         <td>0.7158</td>
+         <td>0.7143</td>
+         <td>0.6731</td>
+         <td>0.5854</td>
+      </tr>
+      <tr>
+         <th>Mean search time</th>
+         <td>2336 µs</td>
+         <td>2750 µs</td>
+         <td>2597 µs</td>
+         <td>2534 µs</td>
+      </tr>
+      <tr>
+         <th>Compression</th>
+         <td>x1</td>
+         <td>x4</td>
+         <td>x8</td>
+         <td>x12</td>
+      </tr>
+      <tr>
+         <th>Upload & indexing time</th>
+         <td>147 s</td>
+         <td>339 s</td>
+         <td>217 s</td>
+         <td>178 s</td>
+      </tr>
+   </tbody>
+</table>
+
+Product Quantization increases both indexing and searching time. The higher the compression ratio, 
+the lower the search precision. The main benefit is undoubtedly the reduced usage of memory.
+
+#### Arxiv-titles-384-angular-no-filters
+
+<table>
+   <thead>
+      <tr>
+         <th></th>
+         <th>Original</th>
+         <th>1D clusters</th>
+         <th>2D clusters</th>
+         <th>4D clusters</th>
+         <th>8D clusters</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <th>Mean precision</th>
+         <td>0.9837</td>
+         <td>0.9677</td>
+         <td>0.9143</td>
+         <td>0.8068</td>
+         <td>0.6618</td>
+      </tr>
+      <tr>
+         <th>Mean search time</th>
+         <td>2719 µs</td>
+         <td>4134 µs</td>
+         <td>2947 µs</td>
+         <td>2175 µs</td>
+         <td>2053 µs</td>
+      </tr>
+      <tr>
+         <th>Compression</th>
+         <td>x1</td>
+         <td>x4</td>
+         <td>x8</td>
+         <td>x16</td>
+         <td>x32</td>
+      </tr>
+      <tr>
+         <th>Upload & indexing time</th>
+         <td>332 s</td>
+         <td>921 s</td>
+         <td>597 s</td>
+         <td>481 s</td>
+         <td>474 s</td>
+      </tr>
+   </tbody>
+</table>
+
+It turns out that in some cases, Product Quantization may not only reduce the memory usage, 
+but also the search time.
+
+## Good practices
+
```

Comment:
```suggestion

Compared to Scalar Quantization, Product Quantization offers a higher compression rate. However, this comes with considerable trade-offs in accuracy, and at times, in-RAM search speed.

Product Quantization tends to be favored in certain specific scenarios:

- Deployment in a low-RAM environment where the limiting factor is the number of disk reads rather than the vector comparison itself
- Situations where the dimensionality of the original vectors is sufficiently high
- Cases where indexing speed is not a critical factor

In circumstances that do not align with the above, Scalar Quantization should be the preferred choice.

```